packages/client/tests/functional/_utils/setupFilesAfterEnv.ts

```diff
@@ -37,4 +34,33 @@ function sanitizeLineNumbers(message: string) {
   return stripAnsi(message).replace(/^(\s*→?\s+)\d+/gm, '$1XX')
 }
 
+/**
+ * Utility that busts test flakiness by running the same test multiple times.
+ * This is especially useful for tests that fail locally but not on CI.
+ * @param n Number of times to repeat the test
+ * @returns
+ */
+function testRepeat(n: number) {
```

Comment:
Doesn't seem to be used.
Also, what's the difference with `jest.retries(n)`?

Reply:
https://prisma-company.slack.com/archives/C03V1UE750D/p1663711087498559?thread_ts=1663709917.516639&cid=C03V1UE750D

> `testRepeat` repeats a test `n` times and fails if any of the repetitions have failed. It's the opposite of `jest.retryTimes` which will succeed if any has passed.